MH4U Dex/ItemEncyclopediaTableViewController.m

```diff
@@ -81,4 +105,17 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender
     }
 }
 
+#pragma mark - Search Bar Delegate Methods
+
+- (void)searchBar:(UISearchBar *)searchBar textDidChange:(NSString *)searchText
+{
+    if (!searchText.length) {
+        self.fetchedResultsController = [self fetchedResultsControllerWithPredicate:nil];
+    } else {
+        NSPredicate *predicate = [NSPredicate predicateWithFormat:@"%K contains[cd] %@", ItemAttributes.name, searchText];
+        self.fetchedResultsController = [self fetchedResultsControllerWithPredicate:predicate];
+    }
```

Comment:
This one is debatable, but I think I would change the `fetchedResultsControllerWithPredicate:` method to something like `fetchedResultsControllerWithSearchTerm:`, and make that method responsible for building the NSPredicate. That way, the predicate setup is centralized.